geometry_msgs/Transform position
---
float32[] result
